client/components/game/program/program.jsx

```diff
@@ -13,6 +13,7 @@ export default React.createClass({
       <div className="program">
         <div className="program__wrapper">
           <img className="program__image" src={this.props.program.image} />
+            <span>TESTABC</span>
```

Comment:
no I did not. I can remove it when I am home and repush, I do not know how to edit that on github directly.